scripts/go/configs/.golangci.toml

```diff
@@ -11,6 +11,13 @@ min-occurrences = 5
 [linters-settings.exhaustive]
 default-signifies-exhaustive = true
 
+[linters-settings.revive]
+ignore-generated-header = false
+severity = "warning"
+confidence = 3
+errorCode = 0
+warningCode = 0
```

Comment:
I think these two should be removed, unless there's an actual need for them (I don't think there is).

Reply:
Cool, removed them for now 👍